projects/components/src/menu-dropdown/menu-dropdown.component.ts

```diff
@@ -37,4 +40,12 @@ export class MenuDropdownComponent {
 
   @ContentChildren(MenuItemComponent)
   public items?: QueryList<MenuItemComponent>;
```

Comment:
nit: this can be private now

Reply:
It's assigned by angular - unless something changed anything managed by angular has always been public.